Dockerfile

```diff
@@ -7,11 +7,12 @@ USER root
 RUN apk add --update \
     ruby-dev \
     ruby-rake \
-    make \ 
+    make \
     gcc \
-    build-base 
+    build-base
 
 RUN gem install --no-user-install ruby-fifo -v 0.0.1
+RUN fluent-gem install fluent-plugin-map -v 0.1.3
```

Comment:
I believe this is enough to install plugin, no need to copy files below

Reply:
Unfortunately it wasn't enough and it finally worked only when I did both thing but env. where I did it was not very clean. I will double check it today and remove files if possible.

Reply:
@onorua you was right. I tested it from scratch and it worked locally.